access/views.py

```diff
@@ -39,10 +46,125 @@ def index(request):
         })
     return render(request, 'access/ready.html', {
         "courses": courses,
-        "manager": 'gitmanager' in settings.INSTALLED_APPS,
     })
 
 
+def publish(request):
+    """
+    Move a course from store to the main folder
+    """
+    if request.method != "POST":
+        raise Http404()
```

Comment:
HTTP 405 (Method Not Allowed) would be more appropriate, right?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405